Apps/W1/Shopify/app/src/Order handling/Codeunits/ShpfyImportOrder.Codeunit.al

```diff
@@ -299,6 +299,25 @@ codeunit 30161 "Shpfy Import Order"
         end;
     end;
 
+    [NonDebuggable]
+    local procedure ImportCustomAttributtes(ShopifyOrderId: BigInteger; OrderLineId: Guid; JCustomAttributtes: JsonArray)
+    var
+        OrderLineAttribute: Record "Shpfy Order Line Attribute";
+        JToken: JsonToken;
+    begin
+        OrderLineAttribute.SetRange("Order Id", ShopifyOrderId);
+        if not OrderLineAttribute.IsEmpty then
+            OrderLineAttribute.DeleteAll();
+        foreach JToken in JCustomAttributtes do begin
+            Clear(OrderLineAttribute);
+            OrderLineAttribute."Order Id" := ShopifyOrderId;
+            OrderLineAttribute."Order Line Id" := OrderLineId;
+            OrderLineAttribute.Key := JsonHelper.GetValueAsText(JToken, 'key', MaxStrLen(OrderLineAttribute."Key"));
```

Comment:
I think the compiler will still warn although the value isn't overflowing the field.

Reply:
I agree